### PR TITLE
May 29 Update

### PR DESCRIPTION
**Pull request by MugOfStars**
@JiYouMCC Updated with the new genes

### CODE REVIEW

File: js/fr_data.js [hunk 1]
```diff
@@ -81,10 +83,12 @@ FRData.TertiaryGeneList = [
     ['Smirch', 'Limited', '200000 T'],
     ['Scales', 'Limited', '100000 T (Baldwin)'],
     ['Capsule', 'Limited', '110000 T (Baldwin)'],
+    ['Firefly', 'Limited', 'Swipp'],
```
Comment (JiYouMCC) [inline js/fr_data.js]: 'Swipp' change to "Swipp's Swap Stand" is better.

File: js/fr_data.js [hunk 2]
```diff
@@ -29,6 +29,7 @@ FRData.PrimaryGeneList = [
     ['Jaguar', 'Uncommon', '250000 T'],
     ['Ripple', 'Uncommon', '250000 T'],
     ['Jupiter', 'Uncommon', '230000 T'],
+    ['Python', 'Uncommon', 'Coli'],
```
Comment (JiYouMCC) [inline js/fr_data.js]: 'Coli' change to 'The Coliseum' is better. So does line 59

File: lg/basic-zh.json [hunk 3]
```diff
@@ -91,6 +91,7 @@
     "Savannah": "热带草原 Savannah",
     "Jaguar": "美洲豹 Jaguar",
     "Wasp": "黄蜂 Wasp",
+    "Python": "Python",
```
Comment (JiYouMCC) [inline lg/basic-zh.json]: If convenient, you can change this line to

"Python": "蟒蛇 Python",

I can also do it after merge.

File: lg/basic-zh.json [hunk 4]
```diff
@@ -133,6 +135,8 @@
     "Lace": "蕾丝 Lace",
     "Opal": "蛋白石 Opal",
     "Smirch": "蜘蛛网 Smirch",
+    "Filigree": "Filigree",
+    "Firefly": "Firefly",
```
Comment (JiYouMCC) [inline lg/basic-zh.json]: "Firefly": "萤火虫 Firefly",

File: js/fr_data.js [hunk 5]
```diff
@@ -55,6 +56,7 @@ FRData.SecondaryGeneList = [
     ['Saturn', 'Uncommon', '160000 T'],
     ['Hypnotic', 'Uncommon', '185000 T'],
     ['Rosette', 'Uncommon', '185000 T'],
+    ['Morph', 'Uncommon', 'Coli'],
```
Comment (JiYouMCC) [inline js/fr_data.js]: 'Coli' change to 'The Coliseum' is better.

File: lg/basic-zh.json [hunk 6]
```diff
@@ -133,6 +135,8 @@
     "Lace": "蕾丝 Lace",
     "Opal": "蛋白石 Opal",
     "Smirch": "蜘蛛网 Smirch",
+    "Filigree": "Filigree",
```
Comment (JiYouMCC) [inline lg/basic-zh.json]: "Filigree": "掐丝 Filigree",

File: lg/basic-zh.json [hunk 7]
```diff
@@ -114,6 +115,7 @@
     "Bee": "蜜蜂 Bee",
     "Safari": "狩猎 Safari",
     "Rosette": "玫瑰花环 Rosette",
+    "Morph": "Morph",
```
Comment (JiYouMCC) [inline lg/basic-zh.json]: "Morph": "巨蟒 Morph",

**Review by JiYouMCC — CHANGES_REQUESTED** [review]
After change Coli and Swipp, I can approve this pull request. If you are convenient, you can change the chinese translate, or I can do it after merge.

**Comment by MugOfStars** [comment]
@JiYouMCC changed Coli and Swipp, and added in the Chinese translations!

**Comment by JiYouMCC** [comment]
@LordZarcock Thanks!

**Pull request merged**